Source/WebKit/WebProcess/WebCoreSupport/WebFrameLoaderClient.cpp

```diff
@@ -648,6 +656,11 @@ void WebFrameLoaderClient::dispatchDidFailLoad(const ResourceError& error)
 
     // ARRISAPOL-1754: Skip sending renderingStarted event for failed URLs
     m_sendRenderingAfterFailedLoad = true;
+
+    auto errorDescription = error.localizedDescription();
+    auto errorMessage = makeString("Failed to load resource", errorDescription.isEmpty() ? "" : ": ", errorDescription);
```

Comment:
This is redundant - remove it, and replace errorMessage with error.localizedDescription in the log below